tests/integration/dom0-template/usr/sbin/mdadm

```diff
@@ -1,2 +1,6 @@
 #!/bin/sh
-echo -n "$@"
+echo "\
+# This test data is used and expected by tests/unit/test_mdadm_arrays.py to unit-test bugtool.mdadm_arrays()
+# This configuration was auto-generated on 27 Jun 1971 08:00
```

Comment:
Is this date correct? Is it relevant? I would have used `cat <<EOF` to emit something verbatim rather than `echo`.